### ISSUE TITLE
<b>, <br> and <i> tags

### ISSUE THREAD

**Issue by slavaGanzin** [breakdance/breakdance#4]
Hello, thanks for awesome project

```
console.log(breakdance('<b>fail</b><br><i>fail</i><br><strong>success</strong>'))
<b>fail</b><br>
<i>fail</i><br>
**success**
```
As I found playing with markup while writing this issue. `<b> <i> and <br>` works as is and can be not transpiled into markdown `**bold** *italic* \n`. It would be great if this would be documented in breakdance.

Indeed I need this behavior. Of course I can write my own regexps or tune handlers and I'll do so. But I think there should be a option key that change this behaviour

Thanks for you work

**Comment by jonschlinkert** [comment]
Agreed, I'll push up a fix. thanks for the issue

**Comment by jonschlinkert** [comment]
published and released as 2.0.0. I had to do a major since this is a breaking change.

**Comment by slavaGanzin** [comment]
Thanks for your time
